assets/src/application/ui/input/EspressoButton/Tooltip.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Tooltip as DefaultTooltip } from 'antd';
+import { TooltipPropsWithTitle } from 'antd/lib/tooltip';
+import { withViewportMatch } from '@appDisplay/viewport';
+import { __ } from '@wordpress/i18n';
+
+import { isEmpty } from '@appServices/utilities/string';
+import './style.scss';
+
+interface TicketsChainedButtonProps extends TooltipPropsWithTitle {
+	button: JSX.Element;
```

Comment:
```suggestion
	button: React.ReactNode;
```

Reply:
got failing tests and had to revert back, sorry